From 6c8642c9eaba2f4c3699ea502c1469b25ef79085 Mon Sep 17 00:00:00 2001
From: George Chan <gchan9527@gmail.com>
Date: Mon, 24 Jun 2024 20:13:14 +0800
Subject: [PATCH 09/28] media: qcom: camss: Add sc7180 support

Add in functional logic throughout the code to support the sc7180.

Signed-off-by: George Chan <gchan9527@gmail.com>
(cherry picked from commit 2399afb5f52117950512ab19b310a42a70ef9665)
Signed-off-by: Jesse <jesse@jjo.ninja>
---
 drivers/media/platform/qcom/camss/camss-vfe.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/platform/qcom/camss/camss-vfe.c b/drivers/media/platform/qcom/camss/camss-vfe.c
index 20e0d6604..cd2b3dd6a 100644
--- a/drivers/media/platform/qcom/camss/camss-vfe.c
+++ b/drivers/media/platform/qcom/camss/camss-vfe.c
@@ -341,6 +341,7 @@ static u32 vfe_src_pad_code(struct vfe_line *line, u32 sink_code,
 		break;
 	case CAMSS_660:
 	case CAMSS_2290:
+	case CAMSS_7180:
 	case CAMSS_7280:
 	case CAMSS_8x96:
 	case CAMSS_8250:
@@ -1989,6 +1990,7 @@ static int vfe_bpl_align(struct vfe_device *vfe)
 	int ret = 8;
 
 	switch (vfe->camss->res->version) {
+	case CAMSS_7180:
 	case CAMSS_7280:
 	case CAMSS_8250:
 	case CAMSS_8280XP:
